core/src/test/scala/com/phaller/rasync/test/base.scala

```diff
@@ -1579,34 +1571,49 @@ class BaseSuite extends FunSuite {
     pool.onQuiescenceShutdown()
   }
 
-  test("New ImmutabilityLattice: successful joins") {
-    val lattice = Immutability.ImmutabilityLattice
-
-    val mutability1 = lattice.join(Immutable, ConditionallyImmutable)
-    assert(mutability1 == ConditionallyImmutable)
-
-    val mutability2 = lattice.join(ConditionallyImmutable, Mutable)
-    assert(mutability2 == Mutable)
-
-    val mutability3 = lattice.join(Immutable, Mutable)
-    assert(mutability3 == Mutable)
-
-    val mutability4 = lattice.join(ConditionallyImmutable, Immutable)
-    assert(mutability4 == ConditionallyImmutable)
-
-    val mutability5 = lattice.join(Mutable, ConditionallyImmutable)
-    assert(mutability5 == Mutable)
-
-    val mutability6 = lattice.join(Mutable, Immutable)
-    assert(mutability6 == Mutable)
-  }
+  //  test("New ImmutabilityLattice: successful joins") {
```

Comment:
Commented code should not be committed. Either it should be removed entirely, or it should be developed in a separate branch.